ibis/expr/operations.py

```diff
@@ -2220,10 +2220,15 @@ def schema(self):
         names = []
         types = []
 
-        # All exprs must be named
         for e in self.by + self.metrics:
-            names.append(e.get_name())
-            types.append(e.type())
+            if isinstance(e, ir.DestructValue):
```

Comment:
can you add a comment here for future readers

Reply:
Added